runtime/src/accounts_db.rs

```diff
@@ -1311,37 +1303,52 @@ impl AccountsDB {
         accounts_index: &AccountsIndex<AccountInfo>,
         pubkey: &Pubkey,
     ) -> Option<(Account, Slot)> {
-        let (lock, index) = accounts_index.get(pubkey, Some(ancestors), None)?;
-        let slot = lock[index].0;
+        let (slot, store_id, offset) = {
+            let (lock, index) = accounts_index.get(pubkey, Some(ancestors), None)?;
+            let slot_list = lock.slot_list();
+            let slot = slot_list[index].0;
```

Comment:
nits: can't we do like this destructuring?

```rust
let (slot, {store_id, offset, ...}) = slot_list[index];
```

Reply:
yup totally!

Reply:
yup totally, updated!